keymanagementscenarios.md

```diff
@@ -0,0 +1,105 @@
+Key management for container signing can be broadly categorized into three general use cases:
+- Key Setup/Signing (How are keys set up? How are keys accessed by Notary v2 client when signatures are generated?)
+- Trust store configuration (How do runtime environments determine which keys to trust?)
+- Revocation (How do key owners create/revoke keys? How do runtime environments get this information?)
+
+Personas:
+- Publisher: User who builds and signs containers.
+    - Publisher Admin: In some scenarios a publisher will include a group of users (i.e. teams or enterprises). Admin users (i.e. security administrator) will be responsible for configuring roots, provide access to use or generate delegate keys, and make decisions for key revocation.
+    - Publisher Builder [TODO: better wording]: In some scenarios a publisher will include a group of users (i.e. teams or enterprises). Builders (i.e. developers, build hosts) will be responsible for building contatiner images. 
+    - Publisher Singer [TODO: better wording]: In some scenarios a publisher will include a group of users (i.e. teams or enterprises). Signers (i.e. developers, signing hosts) will be responsible for signing container images.
```

Comment:
```suggestion
    - Publisher Signer [TODO: better wording]: In some scenarios a publisher will include a group of users (i.e. teams or enterprises). Signers (i.e. developers, signing hosts) will be responsible for signing container images.
```

Reply:
When an artifact is signed, can we get a song to play? Ding for good, dong for error.